_posts/abaker8_posts/abaker8/2017-05-30-abaker8-pair-program.md

```diff
@@ -0,0 +1,12 @@
+--- 
+layout: post
+author: abaker8
+title: "Amandas Pair Program"
+---
+
+Here is my program. 
+<iframe src="https://trinket.io/embed/python3/3c42b7e687" width="100%" height="356" frameborder="0" 
+
+
+
+So for this code, I struggled in two ways. 1. After going back to my code later in the day, it made me enter the word three times before giving a response. I wish I had more time to step away to look it over. Secondly I was wondering how this could work with a word such as running? The infinitive of the word would be run not runn, but that would be removing the ING. What can one do to ensure you can get the correct root workd reardless of the stem. 
```

Comment:
For the purposes of this exercise, we won't worry about identifying root words.  For more on that, take Text Mining :)

Here's a remix of your trinket with a few of the issues resolved:
https://trinket.io/python3/9b96860a90

You asked for input inside the function, when you didn't need to.  You also didn't print the result of the function call in a place.  Finally, you printed the non-gerund message instead of returning it.  Very close, and you got the logic to test for the gerund and extract the root right.  Nicely done!  With a partner I'm pretty sure you would've been able to resolve these.  Hope you're feeling better!